TESTS/mbed_hal/common_tickers/main.cpp

```diff
@@ -52,6 +52,7 @@ unsigned int ticker_overflow_delta;
 
 /* Auxiliary function to count ticker ticks elapsed during execution of N cycles of empty while loop.
  * Parameter <step> is used to disable compiler optimisation. */
+MBED_NOINLINE
```

Comment:
Why was this inserted?

Reply:
Never mind. Commit b18f690b39d9d1ad825375665b6281fc15ad4a74 explained it perfectly.